api/filter/http_connection_manager.proto

```diff
@@ -1,6 +1,7 @@
 syntax = "proto3";
 
 package envoy.api.v2.filter;
+option go_package = "filter";
```

Comment:
does this become sub package automatically? like api/filter?

Reply:
go files don't have fully qualified names in the package, just the directory in which the sources reside.